requirements.txt

```diff
@@ -6,3 +6,4 @@
 pyyaml>=3.10
 aiohttp>=3.7.4
 setuptools>=39.0
+pillow>=6.2.2
```

Comment:
EL8 uses `python3-pillow-5.1.1-16.el8.x86_64.rpm` so `pillow>=5.1.0` should likely be our minimum cut-off.